docs/farming-&-staking/farming/advanced-cli/cli-prerequisites.mdx

```diff
@@ -62,3 +62,18 @@ If you have a server with no firewall, there is nothing to be done, otherwise ma
 
 On the desktop side if you have a router in front of your computer, you'll need to forward TCP and UDP ports to the machine on which your node is running (how this is done varies from router to router, but there is always a feature like this, refer to [How to Forward Ports](../Additional-Guides/port-forwarding) for a more in-depth tutorial).
 If you're connected directly without any router, then again nothing needs to be done in such case.
+
+### Storage
```

Comment:
What about emphasis on key points that Space Acres has?

Reply:
Oops, got lost on their way. Changed!